Automattic/wp-calypso
# Help Center: Redirect non eligible users to home when accessing wapuu

Related to #93246 
Fixes https://github.com/Automattic/wp-calypso/issues/93246

## Proposed Changes

* Non-eligible users are landing on `wapuu` and getting a blank page, this PR fixes it by sending them to `/home` since they are not eligible to access `wapuu`.

## Why are these changes being made?


* Fix users landing on the help center and getting a blank screen.

<img width="514" alt="image" src="https://github.com/user-attachments/assets/5107ca0f-cdb6-44a1-8beb-dff73376714c">


## Testing Instructions



* Check live link or pull and run this branch
* Add `/help?help-center=wapuu` to the URL, example: `http://calypso.localhost:3000/help?help-center=wapuu`
* Help center should open with wapuu if you're using a paid user (eligible)
* If you're using a free user, it should open the help center on home.


Regressions:

Kris noted here p1723120199029909/1723109797.283309-slack-C02T4NVL4JJ that there's an experiment running on Wapuu with free users, we need to make sure this PR is not interfering with the results of this experiment.

I have investigated, and the experiment uses a flag `&flags=wapuu` to 'choose' users that will get wapuu. To make sure this PR will not interfere with the experiment, you can add this flag too and make sure that free and paid users get Wapuu.

Example: http://calypso.localhost:3000/help?help-center=wapuu&flags=wapuu

@escapemanuele 

## Pre-merge Checklist
